index.html

```diff
@@ -0,0 +1,320 @@
+            <p>JavaScript before ECMAScript 2015 does not have block statement scope; rather, a variable declared within a block is
+                local to the function (or global scope) that the block resides within. For example the following code will log 5,
+                because the scope of x is the function (or global context) within which x is declared, not the block, which in this
+                case
+                is an if statement.</p>
+            <code>if (true) { var x = 5; } console.log(x); // 5</code>
+            <p>This behavior changes, when using the let declaration introduced in ECMAScript 2015.</p>
+            <code>if (true) { let y = 5; } console.log(y); // ReferenceError: y is not
+                    defined</code>
+         </section>
+         <section id="Global_variables"><h1 id="7"> Global variables</h1>
+        <p>Global variables are in fact properties of the global object. In web pages the global object is window, so you can
+            set
+            and access global variables using the window.variable syntax.</p>
+        <p>Consequently, you can access global variables declared in one window or frame from another window or frame by
+            specifying
+            the window or frame name. For example, if a variable called phoneNumber is declared in a document, you can refer to
+            this
+            variable from an iframe as parent.phoneNumber.</p>
+        </section>
+        <section id="Constants">
+            <h1 id="8">Constants</h1>
+            <p>You can create a read-only, named constant with the const keyword. The syntax of a constant identifier is the same as
+                for a variable identifier: it must start with a letter, underscore or dollar sign and can contain alphabetic,
+                numeric,
+                or underscore characters.</p>
+            <code>const PI = 3.14;</code>
+            <p>A constant cannot change value through assignment or be re-declared while the script is running. It has to be
+                initialized to a value.</p><br>
+            <p>The scope rules for constants are the same as those for let block scope variables. If the const keyword is omitted,
+                the
+                identifier is assumed to represent a variable.</p><br>
+            <p>You cannot declare a constant with the same name as a function or variable in the same scope. For example:</p>
+            <code>// THIS WILL CAUSE AN ERROR function f() {}; const f = 5; // THIS WILL
+                    CAUSE AN ERROR ALSO function f() { const g = 5; var g; //statements
+                    }</code>
+            <p>However, object attributes are not protected, so the following statement is executed without problems.</p>
+            <code>const MY_OBJECT = {"key": "value"}; MY_OBJECT.key = "otherValue";</code>
+
+        </section>
+       
+       
+        <section id="Data_types">
+            <h1 id="9">Data types</h1>
```

Comment:
Please make sure to only have on h1 per page as it makes it easier for the search engines to determine the main header of your page. For any other header on your page, you can h2-h6